models/ed/R/model2netcdf.ED2.R

```diff
@@ -166,8 +167,10 @@ model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date,
     PEcAn.logger::logger.info("*** Writing netCDF file ***")
 
     out <- unlist(out_list, recursive = FALSE)
+    #create nc file with slots for all variables
     nc <- ncdf4::nc_create(file.path(outdir, paste(y, "nc", sep = ".")),
                            nc_var)
+    on.exit(ncdf4::nc_close(nc))
```

Comment:
Not sure about this design change over manually closing the files. Doesn't this delay the closing of the file until the exit of the function. Since this is in a loop, seems like we're opening connections to a lot of files at once, which seems inefficient and could lead to a "too many connections" error for a long timeseries. Also, since we're reusing `nc` in the loop, it seems like we're going to loose the connection to all but the last of the files opened.

Reply:
Thanks.  I think keeping these `on.exit()` lines will ensure the file gets closed if there is an error or interruption and they can co-exist with the manual closing of files within this loop.

Reply:
Nevermind, just tested that and it doesn't work like I expected.  I'll just revert this change for now then.